### PR TITLE
GROW-175, GROW-176 Getting started - causes and places

### PR DESCRIPTION
**Pull request by ryan-ludwig**

### CODE REVIEW

File: src/pages/GetStarted/GetStartedCauses.vue [hunk 1]
```diff
@@ -1,31 +1,191 @@
...
+					checked: false,
+				},
+				{
+					id: 4,
+					name: 'education',
+					checked: false,
+				},
+				{
+					id: 0, // TODO: Couldn't find an id for technology
+					name: 'technology',
```
Comment (ryan-ludwig) [inline src/pages/GetStarted/GetStartedCauses.vue:90]: @emuvente Do you know what the id is for "technology" category?

Comment (emuvente) [reply src/pages/GetStarted/GetStartedCauses.vue:90]: I can't find one for htat either, so I think we'll have to make it.

Comment (mcstover) [reply src/pages/GetStarted/GetStartedCauses.vue:90]: I think we need to figure out what the loan search criteria is supposed to be for this. Then if you do create a new Loan Channel, please add it to the query map too (https://github.com/kiva/ui/blob/master/src/plugins/loan-channel-query-map.js)

Comment (emuvente) [reply src/pages/GetStarted/GetStartedCauses.vue:90]: I believe for the technology category it's just the "Technology" tag

File: src/components/Kv/KvCauseSelector.vue [hunk 2]
```diff
@@ -45,7 +45,7 @@
 						alt=""
 					>
 				</div>
-				<div class="kv-cause-selector__text">{{ cause }}</div>
+				<div class="kv-cause-selector__text">{{ capitalizeFirstLetter(cause) }}</div>
```
Comment (eddieferrer) [inline src/components/Kv/KvCauseSelector.vue:48]: I believe you could also use: `{{ cause|changeCase('titleCase') }}`

Comment (ryan-ludwig) [reply src/components/Kv/KvCauseSelector.vue:48]: > I believe you could also use: `{{ cause|changeCase('titleCase') }}`

I think that would change 'COVID-19' to 'Covid-19' unfortunately

Comment (eddieferrer) [reply src/components/Kv/KvCauseSelector.vue:48]: Sign, I was afraid of that.

Comment (eddieferrer) [reply src/components/Kv/KvCauseSelector.vue:48]: @ryan-ludwig since all the causes are one word, we can just use CSS `text-transform: capitalize;`

**Review by mcstover — APPROVED** [review]

**Review by emuvente — APPROVED** [review]

**Pull request merged**